File: C#/aluno.cs
using System;
public class aluno{
	private string nome;
	private int qtd_men;
	public void set_dados(){
		System.Console.WriteLine("Digite o nome do dicente");
		nome=System.Console.ReadLine();

		System.Console.WriteLine("Digite a quantidade  de matérias do dicente");
		qtd_men=int.Parse(System.Console.ReadLine());

	}
	public void print_dados(){
		System.Console.WriteLine(" O nome do dicente{0}",nome);
		System.Console.WriteLine("A quantidade de matérias do dicente{0}",qtd_men);

	}
	public float mensalidade(){
		float men;
        men=qtd_mem*200;
        return men;
	}
}

File: CPP/faculdadef/aluno.cpp
#include "aluno.h"

aluno::aluno()
{

}

void aluno::set_dados(){
    cout <<"Digite o nome do dicente: "<<"\n\n";
    std::getline(cin,nome);
    cout <<"Digite o curso de "<< nome << "\n\n";
    std::getline(cin,curso);
    cout<<"Digite a quantidade de matérias de "<< nome <<"\n\n";
    cin >> qtd_materias;
    cin.ignore();
}
void aluno::print_dados(){
    cout<< "O nome do dicente "<< nome << "\n\n";
    cout<< "O curso de "<< nome << " é "<< curso <<"\n\n";
    cout<<" A quantidade de matérias de "<< nome << " é "<< qtd_materias <<"\n\n";
}
float aluno::mensalidade(){
    float men;
    men= qtd_materias*200;
    return men;
}
string aluno::ret_nome(){
    return nome;
}
